### PR TITLE
add automated link checking

### PR DESCRIPTION
**Pull request by afeld**
The build failures are legitimate – we have broken links between pages! This pull request is to warn us if we create any in the future.

### CODE REVIEW

**Comment by afeld** [comment]
Internal broken links fixed. Will fix the external ones after #38 is merged.

**Comment by afeld** [comment]
Fixed!

```
$ bundle exec rake
bundle exec jekyll build -d _site/before-you-ship/
Configuration file: /Users/aidanfeldman/dev/18F/before-you-ship/_config.yml
            Source: /Users/aidanfeldman/dev/18F/before-you-ship
       Destination: _site/before-you-ship/
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 0.372 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
Running ["ImageCheck", "ScriptCheck", "LinkCheck"] on ./_site on *.html...


Checking 71 external links...
Ran on 16 files!


HTML-Proofer finished successfully.
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,3 +13,18 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
...
+
+```bash
+bundle
+bundle exec jekyll serve
+```
+
+### Running the tests
+
+```bash
+bundle exec rake
```
Comment (DavidEBest) [inline CONTRIBUTING.md]: Should these `bundle exec jekyll` calls be replaced by `./go` commands?

**Comment by afeld** [comment]
Updated!

**Pull request merged**